shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/model/Answer.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.avishek.phong.kmmic.domain.model
+
+interface Answerable {
+    val id: String
+    val content: String?
+    val placeholder: String?
+}
+
+data class Answer(
+    override val id: String,
+    val text: String?,
```

Comment:
Same goes here. How about renaming `text` to `content`?

Reply:
solved in efe216b